src/main/java/com/documaster/rms/noark/ws/client/accesslog/AccessLogResult.java

```diff
@@ -0,0 +1,55 @@
+package com.documaster.rms.noark.ws.client.accesslog;
+
+import java.util.Date;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class AccessLogResult {
+
+	private final String objectUUID;
+	private final String objectType;
+	private final Integer version;
+	private final String userName;
+	private final Date createdDate;
+
+	@JsonCreator
+	private AccessLogResult(
+			@JsonProperty("id") String objectUUID,
```

Comment:
The `JsonProperty` string fields are quite off, Jackson fallbacks to use the actual field name in the class or in the constructor so things magically work even now.
We also need to add the `userId` field I think.

Reply:
Fixed w/ https://github.com/documaster/ws-client-n5-java/pull/44/commits/c7867dfe979cdcbf451bfa02e5f443322e12479a